### DISCUSSION TITLE
Default hair

### DISCUSSION THREAD

**Discussion by romeoDev-sys** [Monika-After-Story/MonikaModDev#10027] [QA - tech support + general]
I was wondering if anyone could give me directions on how to change Monika's default hairstyle, if that is even possible. 
I wanted my Moni to use the short ponytail as default, instead of the normal hair. 
Any help will be much appreciated :)

**Comment by dreamscached** [comment]
It will be unlocked eventually.

**Comment by Booplicate** [comment]
Beat in mind, you can change hair, but you cannot change what default is.

**Comment by ashyashton** [comment]
Well, _theoretically_ you can change the default hair, but it would require editing the files and whatnot. Complicated, and unnecessary as she stays in whatever hair you set her in until you change it, meaning whatever hair you select will be her "default" hair as far as anything important goes. The only way this would be untrue would be if you had that one submod to have her choose her hair and outfits, which I assume you wouldn't have if you were trying to have her stick to one hairstyle.

**Comment by Booplicate** [comment]
No because that may break lots of things.

**Comment by Chamelllemon** [comment]
I guess you're using auto hair change submod

**Comment by ashyashton** [comment]
> No because that may break lots of things.

Which is why I specifically said it's complicated and unnecessary. I'm not sure why you put "no" at the beginning like you're correcting me, when your statement doesn't contradict mine. "Can break things" doesn't mean "impossible." It just means it's really not worth it, which is exactly what I was saying in my own response.

Though I will say, I'm not sure how editing an image file(the type of file editing I meant) would break things. That's probably not technically on-topic, though, so I'll save that question for another day haha
